src/window/index.ts

```diff
@@ -116,12 +116,18 @@ export async function openFirstRun() {
       },
     });
 
+  if (!window) {
```

Comment:
I'm thinking that we might want to log an error if we can't get a window for any reason